resources/github-actions.md

```diff
@@ -45,6 +45,59 @@ There is a link provided at the end of every C++ and Python build job where the
 
 ![](/assets/images/downloads.png)
 
+## Using CI Artifacts in Other PRs
+
+For changes that cross library boundaries, it may be necessary to test a pull request to one library with changes from a pull request to another library.
+To do this, it is necessary to download CI artifacts (described in the above section) from one library during the CI workflow of another library.
+First, determine the pull request number and commit hash that need to be tested from the other library.
```

Comment:
suggestion (non-blocking): It might be helpful to explain how the links described in the above section translate to the arguments to `rapids-get-artifact` below.